mbuild/formats/lammpsdata.py

```diff
@@ -762,23 +769,25 @@ def _get_dihedral_types(
     if use_rb_torsions:
         unique_dihedral_types = OrderedDict(
             enumerate(
-                [
-                    (
-                        round(dihedral.type.c0 * lj_unit, 5),
-                        round(dihedral.type.c1 * lj_unit, 5),
-                        round(dihedral.type.c2 * lj_unit, 5),
-                        round(dihedral.type.c3 * lj_unit, 5),
-                        round(dihedral.type.c4 * lj_unit, 5),
-                        round(dihedral.type.c5 * lj_unit, 5),
-                        round(dihedral.type.scee, 1),
-                        round(dihedral.type.scnb, 1),
-                        dihedral.atom1.type,
-                        dihedral.atom2.type,
-                        dihedral.atom3.type,
-                        dihedral.atom4.type,
-                    )
-                    for dihedral in structure.rb_torsions
-                ]
+                OrderedSet(
+                    *[
+                        (
+                            round(dihedral.type.c0 * lj_unit, 5),
```

Comment:
Something that might be helpful in general, in case we ever need to change the precision of these writers would be to put the various decimal amounts into `dihedral_precision` variables. That way we only would have to update one or two variables instead of 5+

Reply:
I like that change, probably as good as any of a spot to include this in this PR.